tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -273,18 +230,10 @@ def main():
         help="threads count for loading zips",
     )
 
-    parser.add_argument(
-        "-o",
-        "--omd_api_key",
-        default="",
-        help="user key for working with openmobilitydata API",
-    )
-
     # Required in order to use Transitlands api
     parser.add_argument(
         "-T",
         "--transitland_api_key",
-        required=True,
```

Comment:
Why it was required before? What happens now if it is specified or not specified?

Reply:
As per the earlier implementation, Transit Land API key was always necessary. I think this gives more flexibility to just crawl just Mobility without the hassle of generating a Transit Land API key.

If the --source is set to 'all' or 'transitland' : the key is required. If it is not specified, line 255 will return an error.

If the --source is set to 'mobilitydb' : the key is not required and it will crawl just Mobility DB. Specifying the Transit Land key wouldn't matter since it is not being used.